nestkernel/conn_builder.cpp

```diff
@@ -114,6 +116,19 @@ nest::ConnBuilder::ConnBuilder( Network& net,
       : ConnParameter::create( ( *syn_defaults )[ names::delay ], net_.get_num_threads() );
   }
   register_parameters_requiring_skipping_( *delay_ );
+  // Structural plasticity parameters
+  // Check if both pre and post synaptic element are provided
+  if ( syn_spec->known( names::pre_synaptic_element )
+    && syn_spec->known( names::post_synaptic_element ) )
```

Comment:
Should we raise an error if only one if pre and post is given? The user probably intended to do something and just forgot to provide complete information. Silently doing nothing in that case could lead to hard-to-detect errors or even incorrect results.